### DISCUSSION TITLE
Flashing difficulties with wt32-eth01

### DISCUSSION THREAD

**Discussion by Mitko939** [arendst/Tasmota#13386] [Q&A] [answered]
Hi,
I have a problem to flash wt32-eth01 (v1.2) with tasmota. I tryed the tasmota web installer on Chrome, but i haven't find a tasmota32 version. I try with a simple tasmota version and for one time it works. I had rebooted the wt-32 but i can't find it like AP with wifi.
I conncet from the FT232TL to W32-eth01
GND-->GND
3.3v-->3.3v
TXD-->TX0
RXD-->TX0
and only on the power on of the device the w32-eth01 GND to IO0.
I tryed much time to flash with tasmota, but now it's impossible to put the tasmota on wt32-eth01. Some advice?

I tryed flash via esp_flasher, but on mac the flasher not works ( when i open the program, this turn off itself).
What can i do? Thank you.

**Comment by sfromis** [comment, marked as answer]
Swap tx and rx. It does not work if both tries to transmit on the same pins.

Note that FT232TL is generally not a good one for flashing, as it cannot deliver sufficient power for reliable flashing. And ESP32 takes more power than ESP8266.

**Reply by Mitko939** [reply]
thanks for the quick answer. I was wrong to write. links are like that:
GND-->GND
3.3v-->3.3v
TXD-->RX0
RXD-->TX0

Do you raccomand to try to power the WT32 externaly with 5V when i flash it?

**Reply by ascillato2** [reply]
5V will burn it. Do Not apply 5V to a 3.3V device. Please, use 3.3V in a 3.3V device.

**Reply by Mitko939** [reply]
the wt32-eth01 has 3.3v port and the 5v port. I powered it with 5v on 5v port. On the manual it says that you can power the wt32-eth01 with 3.3v on the pin 3.3v or with 5v on the pin 5v. 
this is the manual.
http://www.wireless-tag.com/wp-content/uploads/2020/08/WT32-ETH01规格书V1.2EN（英文）.pdf

**Comment by ollyins** [comment]
Hi, I also have a problem to find the correct firmware...while reading the closed post #9496 i tried to flash via webinstaller the latest tasmota versio but finally had o ethernet config.
So i have a tasmota on the W32-S1, but need now to flash the correct one!
I need the neopool packages on it.. so i think i have to compile it in some way... - i already did it last year for normal esp32... - have to enter again how i did it. i think i did it with gitpod... so i have to find a working wt32 tasmota and integrate neopool... maybe someone has some hints...

**Reply by sfromis** [reply]
The Ethernet support is by default included in ESP32 builds, without you needing to do anything to include it when you build. You still need to configure gpio pins and a few settings to match the hardware.
